.github/workflows/deploy-storefrontcloud.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           initial_status: in_progress
       - name: Deploy on Storefrontcloud.io
         run: |
-          if curl -s -u ${{ secrets.DOCKER_USERNAME }}:${{ secrets.DOCKER_PASSWORD }} -H 'Content-Type: application/json' -X POST -d '{"code":"${{ env.INSTANCE_CODE }}","frontContainerVersion":"${{ github.sha }}"}' https://farmer.storefrontcloud.io/instances | grep -q '{"code":200,"result":"Instance updated!"}'; then
+          if curl -s -u ${{ secrets.CLOUD_USERNAME }}:${{ secrets.CLOUD_PASSWORD }} -H 'Content-Type: application/json' -X POST -d '{"code":"${{ env.INSTANCE_CODE }}","region":"europe-west1.gcp","frontContainerVersion":"${{ github.sha }}"}' https://farmer.storefrontcloud.io/instances | grep -q '{"code":200,"result":"Instance updated!"}'; then
```

Comment:
Please use Headers instead of BasicAuth
```
curl -H "X-User-Id: CLOUD_USERNAME" -H "X-Api-Key: CLOUD_PASSWORD" -H 'Content-Type: application/json' -X POST -d "{\"code\":\"${{ env.INSTANCE_CODE }}\", \"region\":\"europe-west1.gcp\", \"frontContainerVersion\":\"${{ github.sha }}\"}" https://farmer.storefrontcloud.io/instances

```